cli/types/index.d.ts

```diff
@@ -3499,6 +3499,24 @@ declare namespace Cypress {
      * @see https://on.cypress.io/assertions
      */
     (chainer: 'contain', value: string): Chainable<Subject>
+    /**
+     * Assert that at least one element of the selection is focused, using `document.activeElement`.
+     * @example
+     *    cy.get('#result').should('have.focus')
+     *    cy.get('#result').should('be.focused')
+     * @see http://chaijs.com/plugins/chai-jquery/#containtext

```

Comment:
I know we reference `chai-jquery` here but really these are our own and we should think about fully documenting them and reference our own docs. We actually deviate a little bit from `chai-jquery` even though we took them initially as inspiration.